lib/public/views/Environments/ActiveColumns/environmentsActiveColumns.js

```diff
@@ -29,21 +29,30 @@ export const environmentsActiveColumns = {
         title: true,
         format: (environment) => infoLoggerButtonGroup({ environmentId: environment }),
     },
+    runs: {
+        name: 'Runs',
+        visible: true,
+        sortable: false,
+        size: 'w-10',
+        title: true,
+        format: formatRunsList,
+        balloon: true,
+    },
     updatedAt: {
         name: 'Updated At',
         visible: true,
         sortable: false,
         title: true,
-        size: 'w-15',
-        format: (timestamp) => formatTimestamp(timestamp),
+        size: 'w-10',
+        format: (_, environment) => formatEnvironmentDates(environment.updatedAt),
```

Comment:
Why replacing `timestamp` by `(_, environemnt)` to in the end get the same property?